flexmeasures/data/services/sensors.py

```diff
@@ -64,7 +64,8 @@ def _get_sensor_bdf(sensor: Sensor, staleness_search: dict) -> BeliefsDataFrame
     """Get bdf for a given sensor with given search parameters."""
     bdf = TimedBelief.search(
         sensors=sensor,
-        most_recent_events_only=True,
+        most_recent_beliefs_only=False,
```

Comment:
If we change it like this, we should update the docstring accordingly, and maybe also the util function name, it's too generic.